src/UserInput/Button.react.tsx

```diff
@@ -106,14 +114,13 @@ const Button: React.FC<ButtonProps> = (props: ButtonProps) => {
     ],
     buttonStyle: [
       Styles.background,
-      GlobalStyles.shadow,
       getBackgroundStyle(),
-      props.noGrow ? { width: undefined } : {},
+      props.noGrow ? { ...Spacing.paddingVertical, ...Spacing.largePaddingHorizontal, width: undefined } : {},
       props.nav ? { borderRadius: 19, height: 40 } : {},
       props.buttonStyle,
     ],
-    disabledStyle: props.secondary ? Styles.secondaryDisabledBackground : Styles.primaryDisabledBackground,
-    disabledTitleStyle: props.secondary ? {color: Colors.RED_200} : { color: Colors.WHITE },
+    disabledStyle: props.rank && props.rank in ["secondary", "tertiary"] ? Styles.secondaryDisabledBackground : Styles.primaryDisabledBackground,
+    disabledTitleStyle: props.rank && props.rank in ["secondary", "tertiary"] ? { color: Colors.RED_200 } : { color: Colors.WHITE },
```

Comment:
would checking for a primary rank be simpler than `props.rank in ["secondary", "tertiary"]?

it strikes me that primary is the default rank both here and in `getBackgroundStyle`. we might benefit from resolving that default early and eliminating the null case forever after.